.docker/Dockerfile72

```diff
@@ -0,0 +1,9 @@
+MAINTAINER Lochemem Bruno Michael <lochbm@gmail.com>
+
+RUN apt-get install libcurl4-openssl-dev && \
+    git clone https://github.com/ace411/couchdb_ext.git /usr/local/src/couchdb_ext && \
+    cd /usr/local/src/couchdb_ext && \
+    phpize && ./configure --enable-couchdb_ext CFLAGS="-lcurl -lpthread" && \
+    make && sudo make install && \
+    echo extension=couchdb_ext > /etc/php/7.2/cli/conf.d/couchdb_ext.ini && \
+    rm -rf /usr/local/src/couchdb_ext
```

Comment:
The one thing I'm missing here is what these docker images are based on? The rest looks fine

Reply:
@WyriHaximus, this Docker image is moved to this [repository](https://github.com/peter279k/docker_couchdb_ext) now.

I think these Dockerfiles will be removed from this repository soon :).

Reply:
And it's related to issue #9.

Reply:
@peter279k question still stands for this Docker image ;).

Reply:
I specify them basd on `Ubuntu 18.04` distribution and it's available on [this](https://github.com/peter279k/docker_couchdb_ext).